<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DlgInitial</class>
 <widget class="QDialog" name="QG_DlgInitial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>19</number>
     </property>
     <item>
      <widget class="QLabel" name="lImage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="pixmap">
        <pixmap>image0</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="lWelcome">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;font size=&quot;+1&quot;&gt;&lt;b&gt;Welcome to LibreCAD&lt;/b&gt;
&lt;/font&gt;
&lt;br&gt;
Please choose the unit you want to use for new drawings and your preferred language.&lt;br&gt;
(You can changes these settings later.)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>14</number>
         </property>
         <property name="topMargin">
          <number>14</number>
         </property>
         <property name="rightMargin">
          <number>14</number>
         </property>
         <property name="bottomMargin">
          <number>14</number>
         </property>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbLanguageCmd"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbLanguage"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbUnit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lUnit">
           <property name="text">
            <string>Default Unit:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lLanguage">
           <property name="text">
            <string>GUI Language:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lCmdLanguage">
           <property name="text">
            <string>Command Language:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbUnit</tabstop>
  <tabstop>cbLanguage</tabstop>
  <tabstop>cbLanguageCmd</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QG_DlgInitial</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
